src/storage.ts

```diff
@@ -4,10 +4,7 @@
 
 import {FetchError} from './errors';
 
-// TODO(danielrozenberg): replace this with a storage location that serves the
-// raw compiled binaries without any of the modifications that the Google AMP
-// CDN performs.
-const STORAGE_BASE_URL = 'https://cdn.ampproject.org';
+const STORAGE_BASE_URL = 'https://storage.googleapis.com/org-cdn/org-cdn/rtv/';
```

Comment:
What's up with the double `org-cdn/org-cdn/`?

Reply:
Poor planning - I called GCP storage bucket `org-cdn` and it contains the results of running `amp release`, which produces `org-cdn/rtv/...` and `net-wildcard/...` subfolders

Reply:
Once we switch to Cloudflare R2 it won't matter :)